packages/block-editor/src/components/warning/style.scss

```diff
@@ -14,7 +14,7 @@
 		line-height: $default-line-height;
 		font-family: $default-font;
 		font-size: $default-font-size;
-		color: $gray-900;
+		color: $gray-900 !important;
```

Comment:
Don't merge before removing this.

Reply:
Why you needed `!important` here? I tested with TT1-blocks and wasn't affecting anything?

Reply:
By sheer luck, I was using a Group block with a dark background colour, and that is how I noticed that the `.editor-styles-wrapper .has-background ...` rules override Warning's colour:

`color: currentColor` | `color: $gray-900`
-|-
<img width="300" alt="Captura de ecrã 2021-01-22, às 09 50 40" src="https://user-images.githubusercontent.com/150562/105475550-60df5c00-5c97-11eb-8231-256b041ebdb7.png">|<img width="300" alt="Captura de ecrã 2021-01-22, às 09 50 14" src="https://user-images.githubusercontent.com/150562/105475536-5de46b80-5c97-11eb-9b73-d1384ade6f20.png">

Reply:
I also added `background` in a group that contains nested reusables and still the color seemed fine 🤔 
I also changed the `color settings` and yet no problem.. What theme are you using?

Reply:
I'm using _Seedlet (Blocks)_, which hasn't been updated in a while, so that may be the root issue. Thanks for testing with TT1! I'll remove this patch now.